flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserRowResolver.java

```diff
@@ -170,6 +206,12 @@ public boolean hasTableAlias(String tabAlias) {
     public ColumnInfo get(String tabAlias, String colAlias) throws SemanticException {
         ColumnInfo ret = null;
 
+        if (!isExprResolver && isAmbiguousReference(tabAlias, colAlias)) {
+            String tableName = tabAlias != null ? tabAlias : "";
+            String fullQualifiedName = tableName + "." + colAlias;
```

Comment:
If `tableName` is null, the qualified name is `.colAlias`. Would be better to eliminate the `.` if tableName is null?

Reply:
Yes, I think it's better to eliminate the `.`.

> Could you add more tests mixed with upper case and lower case? E.g. table with upper case columns and reference a column with the same column name but in lower case.

I change the column name `i` of table `bar` to `I` in `HiveDialectQueryITCase` so that the existing test can cover such cases.